### ISSUE TITLE
Linting seed-element

### ISSUE THREAD

**Issue by MeTaNoV** [PolymerElements/seed-element#79]
Using `polylint` on the `index.html` brings the following errors:
```
../iron-doc-viewer/iron-doc-viewer.html:67:5
    Property '_properties' bound to attribute 'hidden$' not found in 'properties' for element 'iron-doc-viewer'
../iron-doc-viewer/iron-doc-viewer.html:69:7
    Property '_properties' bound to attribute 'items' not found in 'properties' for element 'iron-doc-viewer'
../iron-doc-viewer/iron-doc-viewer.html:69:7
    Property '_properties' bound to attribute 'hidden$' not found in 'properties' for element 'iron-doc-viewer'
../iron-doc-viewer/iron-doc-viewer.html:74:5
    Property '_methods' bound to attribute 'hidden$' not found in 'properties' for element 'iron-doc-viewer'
../iron-doc-viewer/iron-doc-viewer.html:76:7
    Property '_methods' bound to attribute 'items' not found in 'properties' for element 'iron-doc-viewer'
../iron-doc-viewer/iron-doc-viewer.html:81:5
    Property '_events' bound to attribute 'hidden$' not found in 'properties' for element 'iron-doc-viewer'
../iron-doc-viewer/iron-doc-viewer.html:83:7
    Property '_events' bound to attribute 'items' not found in 'properties' for element 'iron-doc-viewer'
../iron-doc-viewer/iron-doc-viewer.html:88:5
    Property '_behaviors' bound to attribute 'hidden$' not found in 'properties' for element 'iron-doc-viewer'
../iron-doc-viewer/iron-doc-viewer.html:90:7
    Property '_behaviors' bound to attribute 'items' not found in 'properties' for element 'iron-doc-viewer'
../iron-ajax/iron-ajax.html:38:1
    Property UNKNOWN not found in 'properties' for element 'iron-ajax'
../iron-doc-viewer/iron-doc-property.html:34:9
    Property 'return' bound to attribute 'hidden$' not found in 'properties' for element 'iron-doc-property'
../iron-doc-viewer/iron-doc-property.html:26:82
    Property _paramText not found in 'properties' for element 'iron-doc-property'
../paper-ripple/paper-ripple.html:163:1
    Property isAttached not found in 'properties' for element 'paper-ripple'
```

**Comment by MeTaNoV** [comment]
@garlicnation something for you? :)

**Comment by ronnyroeller** [comment]
+1
